static/api/extensions/redhat/hummingbird/0.1.1/README.md

```
@@ -0,0 +1,11 @@
+# Hummingbird Extension for Podman Desktop
+
+This extension adds a catalog of Hummingbird alternative images to Podman Desktop.
+
+### Features
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)